cmd/tests/cmd_cloud_run_test.go

```diff
@@ -0,0 +1,248 @@
+package tests
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.k6.io/k6/cloudapi"
+	"go.k6.io/k6/cmd"
+	"go.k6.io/k6/lib/fsext"
+	"go.k6.io/k6/lib/testutils"
+)
+
+func TestCloudRunNotLoggedIn(t *testing.T) {
```

Comment:
The test cases below are copies of the ones that `cmd_cloud_test.go` with the running argument difference. What if we turn them into the constructors for the period of deprecation and just run the old/new commands accordingly? 

There are a couple of rationales behind that suggestion. The most obvious is avoiding duplication, keeping the git history more conventional, and consolidating the test case.

The alternative to that is the wish to drop the comment at least near the old tests with referring that at some point it will be dropped in favor of the new one, which should help for future maintenance of these parts of the code

Reply:
Yeah, I agree, I suggested the same about the command itself, if possible we should wrap and do toggles as much as we can. Otherwise, there is also the risk of having a bug somewhere and fixing only one part of the code.

Reply:
Agree with you folks, let's do it 🙇🏻